src/pages/productpage/sortandfilters/sortByPrice.js

```diff
@@ -0,0 +1,11 @@
+function sortByPrice(sortBy, data) {
+  
+   if (sortBy === "HIGH_TO_LOW")
+     return [...data].sort((a, b) => b["price"] - a["price"]);
+ 
+   if (sortBy === "LOW_TO_HIGH")
+     return [...data].sort((a, b) => a["price"] - b["price"]);
```

Comment:
I like how you have made sortByPrice as a  pure function

Reply:
will work on this surya .thank you